content/200-orm/300-prisma-migrate/300-workflows/10-seeding.mdx

```diff
@@ -344,6 +346,101 @@ psql file.sql
 
 </TabbedContent>
 
+### Seeding your database with <inlinecode>@snaplet/seed</inlinecode>
+
+[`@snaplet/seed`](https://docs.snaplet.dev/seed/getting-started/overview) offers a toolkit designed to understand your database schema,
+enabling you to generate production-accurate data and efficiently seed your database.
+
+It supports PostgreSQL and SQLite.
+
+1. Begin by setting up a local database using `npx prisma migrate dev`. For instance, consider the following Prisma schema featuring a `User` and `Post` model:
+
+   ```prisma file=schema.prisma
+   model User {
+     id    Int    @id @default(autoincrement())
+     email String @unique
+     name  String
+     posts Post[]
+   }
+   
+   model Post {
+     id        Int     @id @default(autoincrement())
+     title     String
+     content   String
+     published Boolean
+     user      User    @relation(fields: [userId], references: [id])
+     userId    Int
+   }
+   ```
+
+2. Execute `npx @snaplet/seed init` to introspect your database. This command generates the seed client and creates a `seed.mts` file.
+3. Utilize the generated `@snaplet/seed` client within the `seed.mts` file to compose your seeds.
+
+Here's how to create new users and posts by modifying the initially generated `seed.mts` file:
+
+```ts file=seed.mts highlight=14-20;normal
+/**
+ * ! Executing this script will delete all data in your database and seed it with 10 versions.
+ * ! Make sure to adjust the script to your needs.
+ * Use any TypeScript runner to run this script, for example: `npx tsx seed.mts`
+ * Learn more about the Seed Client by following our guide: https://docs.snaplet.dev/seed/getting-started
+ */
+import { createSeedClient } from '@snaplet/seed'
+
+const seed = await createSeedClient()
+
+// Truncate all tables in the database
+await seed.$resetDatabase()
+
+// Seed the database with 10 User
+await seed.User((x) =>
+  x(10, () => ({
+    // Create 10 posts for each of those users
+    Post: (x) => x(10),
+  }))
+)
+
+process.exit()
+```
+
+4. Add `tsx` as a development dependency:
```

Comment:
It's not a requirement, maybe we should suggest people use any typescript runner available and use `npx tsx` as an example?

Reply:
added a mention to `any typescript runner`